### DISCUSSION TITLE
Sample project using MvvmCross 9?

### DISCUSSION THREAD

**Discussion by ehuna** [MvvmCross/MvvmCross#4529] [Q&A] [answered]
I saw the announcement at https://www.mvvmcross.com/mvvmcross-9.0.0-release/ with .NET 6 and MAUI support - awesome, thanks!

Is there a sample project that uses MvvmCross 9?  I tried looking at some of the samples in https://github.com/MvvmCross/MvvmCross-Samples but couldn't easily find one.

**Comment by Cheesebaron** [comment, marked as answer]
https://github.com/MvvmCross/MvvmCross/tree/develop/Projects/Playground
